sdk/src/client/bindings/java/lib/src/main/java/org/iota/types/AbstractObjectTwo.java

```diff
@@ -0,0 +1,30 @@
+// Copyright 2023 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+package org.iota.types;
+
+import com.google.gson.Gson;
+import com.google.gson.JsonElement;
+
+import java.util.Objects;
+public abstract class AbstractObjectTwo {
```

Comment:
i dont like this, neither do you i guess. But lets rewrite in a different PR as thatll be bigger scale

Reply:
Yes Thibault also commented on this, I'm not sure which of these two is better and all from client and wallet should use the same at the end